src/v2/ion/i18nTransformer.js

```diff
@@ -87,6 +87,15 @@ const I18N_OVERRIDE_MAPPINGS = {
 
   'select-authenticator-unlock-account.authenticator.okta_email': 'oie.email.label',
   'select-authenticator-unlock-account.authenticator.phone_number': 'oie.phone.label',
+  'select-authenticator-unlock-account.authenticator.duo': 'factor.duo',
+  'select-authenticator-unlock-account.authenticator.google_otp': 'oie.google_authenticator.label',
+  'select-authenticator-unlock-account.authenticator.okta_verify.push': 'oie.okta_verify.push.title',
+  'select-authenticator-unlock-account.authenticator.okta_verify.signed_nonce': 'oie.okta_verify.signed_nonce.label',
+  'select-authenticator-unlock-account.authenticator.okta_verify.totp': 'oie.okta_verify.totp.title',
+  'select-authenticator-unlock-account.authenticator.rsa_token': 'factor.totpHard.rsaSecurId',
+  'select-authenticator-unlock-account.authenticator.security_question': 'oie.security.question.label',
+  'select-authenticator-unlock-account.authenticator.symantec_vip': 'factor.totpHard.symantecVip',
```

Comment:
@SamSanjabi-okta  please let me know what authenticators are supported for account unlock.  

as far as I can tel it looks like everything except password, OV totp and fastpass are excluded. AM I correct ?

Reply:
@magizh-okta I'm not sure what these strings are used for, but like SSPR account unlock has possibly 2 verification steps:
1. Primary factors --> has to be one of email, phone or OV push
2. Step Up --> can be either nothing (i.e. no stepUp required), just security question (V1 behavior), or any authenticator enrolled by the user that's allowed for SSO

Reply:
Aha, I see these refer to the remediation forms.  In that case yeah I think this remediation is only used for primary factors during SSU, in which case only email, phone and OV push are needed.